src/Query/Processor/DefaultParamDefinition.php

```diff
@@ -89,6 +89,11 @@ public static function buildParamDefinitions( $vars, $context = null, ResultPrin
 			'values' => [ 'all', 'subject', 'none' ],
 		];
 
+		$params['prefix'] = [
+			'default' => 'none',
+			'values' => [ 'all', 'subject', 'none', 'auto' ],
+		];
```

Comment:
What is the rationale for adding the new parameter to all result formats? As far as I can tell this PR only adds handling of the parameter to a few formats. (I did not check how ValueTextsBuilder is used so might be wrong here.) Having the parameter in all formats but ignored in most seems odd from a product perspective.

Reply:
From a user perspective, we want these options in as many formats as possible. Maps is the only one I am aware of that already has this. If it means touching every format to bring this feature to is, we will try this, because I think it is important to be able to control the namespace behaviour. 
Thanks for reviewing, btw!

Reply:
Yeah, but do users want the parameter to be there even if it is not actually taken into account?

Reply:
No. But would not doing it this way then require changes to SMW core again if we implement it in any other result format? Then it might be better having this in and just adding the availability to the documentation of those formats where it has been implemented.

Reply:
Each result format defines what parameters it supports. There where no changes in SMW core needed for Maps to support an extra parameter.

Reply:
for now the new parameter is only used for "table" and "list" formats. However I see, is it sufficient to declare it using the method `getParamDefinitions` in the specific printer, right ? 
Then, @krabina we could do the same declaration as we add the new parameter to other formats

Reply:
> However I see, is it sufficient to declare it using the method getParamDefinitions in the specific printer, right ?

Yes